src/test/java/com/puppycrawl/tools/checkstyle/DefaultLoggerTest.java

```diff
@@ -54,7 +55,8 @@ public void tearDown() {
     public void testCtor() {
         final OutputStream infoStream = new ByteArrayOutputStream();
         final ByteArrayOutputStream errorStream = new ByteArrayOutputStream();
-        final DefaultLogger dl = new DefaultLogger(infoStream, OutputStreamOptions.CLOSE,
+        final DefaultLogger dl = new DefaultLogger(infoStream,
+                OutputStreamOptions.CLOSE,
```

Comment:
this is formatting, we can keep code as before.

Reply:
Done.